client/belga/image/belga-coverage-editor.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {gettext} from 'superdesk-core/scripts/core/utils';
+import {DropZone} from 'superdesk-core/scripts/core/ui/components/drop-zone';
+import {IEditorComponentProps} from 'superdesk-core/scripts/apps/fields';
+import BelgaCoverageInfo from './belga-coverage-info';
+import BelgaCoverageImages from './belga-coverage-images';
+
+const ALLOWED_TYPES = [
+    'application/superdesk.item.graphic',
```

Comment:
Which common functionality do we need from Superdesk core? I'd like the client repositories to be as independent as possible and it would be good if we could not depend on things where it can be avoided. Can we use some custom type specific to coverages?

Reply:
it gets belga images from search provider where this mimetype is set